compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -17762,15 +17762,29 @@ TR::Node *fmaxminSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier *
          fmin = fmax = secondChild->getFloat();
       else
          {
-         if (firstChild->getFloat() <= secondChild->getFloat())
+         float first = firstChild->getFloat();
+         float second = secondChild->getFloat();
+         int32_t firstBits = *reinterpret_cast<int32_t *>(&first);
+         int32_t secondBits = *reinterpret_cast<int32_t *>(&second);
+         int32_t nZeroBits = 0x80000000;
```

Comment:
We do not need to use variable to hold this values (bits representation of negative and positive 0.0F).